core/storage/topic/gae_models.py

```diff
@@ -311,6 +311,9 @@ class TopicSummaryModel(base_models.BaseModel):
     # uncategorized).
     total_skill_count = (
         datastore_services.IntegerProperty(required=True, indexed=True))
+    # The total number of published chapters in the topic.
+    total_published_node_count = (
+        datastore_services.IntegerProperty(required=True, indexed=True))
```

Comment:
Do we need to default it if it's now being required when there are also existing models that don't yet contain this property?

Also, why does this need to be indexed?

Reply:
Re: indexed. I saw that all of the other `TopicSummaryModel` attributes had `indexed=True`, so I followed suit. Will follow your recommendation here -- do you think it shouldn't be indexed?

Reply:
@BenHenning In general it's fine to index all non-json/text fields. We haven't seen a downside to doing this.

Reply:
It seems the only potential downside is increased storage costs for unused indexes. I don't feel strongly about this, though.